WIP-CHANGELOG.md

```diff
@@ -25,6 +25,9 @@
 - node exporter to v1.2.2 [full changelog](https://github.com/prometheus/node_exporter/blob/master/CHANGELOG.md)
 - updated metrics-server helm chart to version 0.5.2, this upgrades metrics-server image to 3.7.0 [#702](https://github.com/elastisys/compliantkubernetes-apps/pull/702)
 - Updated Dex chart to v0.6.3, and Dex itself to v2.30.0
+- Updated Blackbox chart to v5.3.1, and blackbox app to v0.19.0
+  - HTTP probe: no_follow_redirects has been renamed to follow_redirects
```

Comment:
Will you add this to `release notes` section as well since some users might use this, so I think that it should be clear.